velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -93,7 +110,30 @@ class SubstraitVeloxPlanConverter {
     return lengths_;
   }
 
+  /// Used to find the function specification in the constructed function map.
```

Comment:
nit: Consider clarifying: "Looks up a function by ID and returns function name if found. Throws if function with specified ID doesn't exist."

Reply:
Fixed.